src/modules/exam/exam.controller.ts

```diff
@@ -0,0 +1,107 @@
+import {
+  Controller,
+  Get,
+  Param,
+  UseInterceptors,
+  ParseUUIDPipe,
+  Post,
+  Body,
+  Patch,
+  Delete
+} from '@nestjs/common';
+import { LoggingInterceptor } from 'src/interceptors/logging.interceptor';
+import { UUID } from 'crypto';
+import { ApiTags } from '@nestjs/swagger';
+import { ExamService } from './exam.service';
+import { CreateExamDto } from './dto/create-exam.dto';
+import { UpdateExamDto } from './dto/update-exam.dto';
+
+@ApiTags('Exams')
+@Controller('exams')
+@UseInterceptors(LoggingInterceptor)
+export class ExamController {
+  constructor(private readonly examService: ExamService) {}
+
+  @Post()
+  async create(@Body() createExamDto: CreateExamDto) {
+    try {
+      await this.examService.create(createExamDto);
+
+      return {
+        success: true,
+        message: 'Exam Created Successfully'
+      };

```

Comment:
- create a common response for both success and failed request
- the format will be
```
{
  success: bool
  data: any | null
  message: string
}
```

Reply:
@nguyennhatninh this good for now, but you can create a class or object, that name `JsonResponse`, which have attribute success, data, message, so that other can use that through out project. That is is common response